docs/operate/get-started/other-hardware/manage-modules.md

```diff
@@ -352,8 +370,8 @@ To transfer ownership of a module from one organization to another:
 1. Select the new organization from the dropdown menu, then click **Transfer module**.
 
 1. (Recommended) Transfer the GitHub repository containing the module code to the new owner.
-   Be sure to remove the existing secrets from the repository’s settings before transferring.
-   If the repository is using Viam’s cloud build, the secrets contain an organization API key that will be exposed to the new owner after the repository transfer.
+   Be sure to remove the existing secrets from the repository's settings before transferring.
```

Comment:
:+1: Thanks for this! I do like using ASCII instead of weird punctuation I can't type with my keyboard.